backoffice/backoffice/workflows/airflow_utils.py

```diff
@@ -158,9 +158,14 @@ def restart_workflow_dags(workflow_id, workflow_type, params=None):
     """
     executed_dags_for_workflow = find_executed_dags(workflow_id, workflow_type)
 
-    for dag_id in executed_dags_for_workflow:
+    data = None
+
+    for dag_id, dag_data in executed_dags_for_workflow.items():
         delete_workflow_dag(dag_id, str(workflow_id))
+        # fetch data from previous drag run
+        if data is None:
```

Comment:
we don't have to explicitly check  do `if data` in python, since it will return `False` if it's empty, None, etc.